### DISCUSSION TITLE
getting error after running npm run serve

### DISCUSSION THREAD

**Discussion by DIWAKARKASHYAP** [ruxailab/RUXAILAB#365] [Q&A] [answered]
![image](https://github.com/ruxailab/RUXAILAB/assets/94478026/14aab65d-2cd5-46f7-a1a9-026757ac0525)

**Comment by sushant4612** [comment]
have you watched the video of setup ?

**Comment by DIWAKARKASHYAP** [comment]
not working , i follow the same step 
![image](https://github.com/ruxailab/RUXAILAB/assets/94478026/72891d19-c566-425c-aed4-50c6aa78dcd3)
![image](https://github.com/ruxailab/RUXAILAB/assets/94478026/ee55ac78-b136-43c8-943c-543c7f34ea18)


nothing show on localhost:8080 and i fill the correct api key

**Reply by sushant4612** [reply, marked as answer]
have you followed all the steps which was shown in this video
https://www.youtube.com/watch?v=dAf4LRxITCc 

if you followed all the steps and still getting the white screen just go trough this discussion chats once, you will find answer to your doubt
https://github.com/orgs/ruxailab/discussions/309

**Comment by DIWAKARKASHYAP** [comment]
thanks

**Reply by aayushrawat** [reply]
hey is your project now running?

**Comment by KarinePistili** [comment]
Hello everybody, I'll be closing this issue as discussions about problems with running the app are being talked about on #309
